public/manifest.json

```diff
@@ -20,6 +20,6 @@
   ],
   "start_url": ".",
   "display": "standalone",
-  "theme_color": "#000000",
-  "background_color": "#ffffff"
+  "theme_color": "#5F9645",
```

Comment:
it's such a small thing, but I love the effect on the browser that comes from adding a theme color!